Sources/NetworkProtection/KeyManagement/NetworkProtectionKeychainStore.swift

```diff
@@ -55,7 +55,8 @@ final class NetworkProtectionKeychainStore {
 
     // MARK: - Keychain Interaction
 
-    func readData(named name: String) throws -> Data? {
+    public func readData(named name: String) throws -> Data? {
+        Logger.networkProtectionKeyManagement.debug("Reading key \(name, privacy: .public) from keychain")
```

Comment:
Nice, the new log entries are handy 🙌

Reply:
Just to clarify, I added logs that I found useful during my VPN debug sessions